src/apicast-cli/cmd/generator.lua

```diff
@@ -1,14 +1,18 @@
-local lfs = require 'lfs'
+local utils_path = require "apicast-cli.utils.path"
+local path = require "pl.path"
+local dir = require "pl.dir"
 
 local create_new_project = function (args, _)
   local opts = args or {}
-  local name = string.gsub((opts.name or 'new-app'):lower(), ' ', '_')
-  local current_dir = lfs.currentdir()
-  local destiny = opts.path or current_dir
+  local name = opts.name or 'new-app'
+  name = string.gsub(name:lower(), ' ', '-')
+  local destiny = opts.path or path.currentdir()
 
-  local raw_app = current_dir .. '/blank-app'
-  local new_app = destiny .. "/" .. name
-  assert(os.execute("cp -a '" .. raw_app .. "' '" .. new_app .. "'"))
+  local raw_app = path.join(utils_path.find(), 'apicast-cli/blank-app')
+  local new_app = path.join(destiny, name)
+  local command = {"cp", "-a", raw_app, new_app}
+  print('Generating new project into ' .. new_app)
+  assert(os.execute(table.concat(command, " ")))
```

Comment:
Agree! Will improve later.
@mikz we may need to extract this kind of things to the utils from apicast (if they exist of course)